sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/AlreadyExistException.scala

```diff
@@ -33,6 +33,9 @@ class TableAlreadyExistsException(db: String, table: String)
 class TempTableAlreadyExistsException(table: String)
   extends AnalysisException(s"Temporary table '$table' already exists")
```

Comment:
You just need to change `Temporary table` to `Temporary view`.

Reply:
+1.

Reply:
How about the class name?  TempTableAlreadyExistsException